.github/workflows/tests.yml

```
@@ -182,14 +182,14 @@ jobs:
       fail-fast: false
       matrix:
         # test lower version (w/ stable conda) and upper version (w/ canary conda)
-        python-version: ['3.8']
+        python-version: ['3.9']
```

Python 3.8 is still our lower bound

```suggestion
        python-version: ['3.8']
```